RecoVertex/AdaptiveVertexFinder/python/inclusiveNegativeVertexing_cff.py

```diff
@@ -0,0 +1,62 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoVertex.AdaptiveVertexFinder.inclusiveCandidateVertexFinder_cfi import *
+from RecoVertex.AdaptiveVertexFinder.candidateVertexMerger_cfi import *
+from RecoVertex.AdaptiveVertexFinder.candidateVertexArbitrator_cfi import *
+
+
+inclusiveCandidateNegativeVertexFinder = inclusiveCandidateVertexFinder.clone(
+   vertexMinAngleCosine = cms.double(-0.95) 
+)
+inclusiveCandidateNegativeVertexFinder.clusterizer.clusterMinAngleCosine = cms.double(-0.5)
+
+candidateNegativeVertexMerger = candidateVertexMerger.clone()
+candidateNegativeVertexMerger.secondaryVertices = cms.InputTag("inclusiveCandidateNegativeVertexFinder")
+
+candidateNegativeVertexArbitrator = candidateVertexArbitrator.clone()
+candidateNegativeVertexArbitrator.secondaryVertices = cms.InputTag("candidateNegativeVertexMerger")
+candidateNegativeVertexArbitrator.dRCut = cms.double(-0.4)
+
+inclusiveCandidateNegativeSecondaryVertices = candidateVertexMerger.clone()
+inclusiveCandidateNegativeSecondaryVertices.secondaryVertices = cms.InputTag("candidateNegativeVertexArbitrator")
+inclusiveCandidateNegativeSecondaryVertices.maxFraction = 0.2
+inclusiveCandidateNegativeSecondaryVertices.minSignificance = 10.
```

Comment:
move inside .clone( arguments

Reply:
Done in https://github.com/cms-sw/cmssw/pull/23206/commits/4e37f178d6a3ee3b5778ba933e82064eaf694aa4